whittle/search_sub_networks.py

```diff
@@ -68,19 +83,41 @@ def setup(
         resume: Path to a checkpoint directory to resume from in case training was interrupted, or ``True`` to resume
             from the latest checkpoint in ``out_dir``. An error will be raised if no checkpoint is found. Passing
             ``'auto'`` will resume from the latest checkpoint but not error if no checkpoint exists.
-        data: Data-related arguments. If not provided, the default is ``litgpt.data.Alpaca``.
+        data: Data-related arguments. If not provided, the default is ``litgpt.data.TinyStories`` or ``litgpt.data.Alpaca`` for fine-tuned models.
         train: Training-related arguments. See ``litgpt.args.TrainArgs`` for details.
         eval: Evaluation-related arguments. See ``litgpt.args.EvalArgs`` for details.
         search: Search-related arguments. See ``whittle.args.SearchArgs`` for details.
         logger_name: The name of the logger to send metrics to.
         seed: The random seed to use for reproducibility.
         access_token: Optional API token to access models with restrictions.
+        param_bins: The parameter bins that limit the sub-network params in the search.
+        objective_1: The name of the first objective to optimize (possible - val_loss, perplexity). Defaults to "val_loss".
```

Comment:
instead of `objective_1` and `objective_2` should we call it `performance_metric` and `efficiency_metric` to make sure that we search for these two things jointly?